ee_plugin/ee_plugin.py

```diff
@@ -88,8 +89,7 @@ def tr(self, message):
         :returns: Translated version of message.
         :rtype: QString
         """
-        # noinspection PyTypeChecker,PyArgumentList,PyCallByClass
-        return QCoreApplication.translate("GoogleEarthEngine", message)
+        return utils.translate(message)
```

Comment:
I like this! Thinking we may want to have some utility function for logging messages so we do it consistently also.